.github/workflows/build-git-installers.yml

```diff
@@ -211,11 +211,22 @@ jobs:
           sed -i -e \
             's/recently_seen="\$(get_recently_seen)"/recently_seen="bogus-to-keep-pestering"/' \
             $b/git-extra/git-update-git-for-windows
-      - name: Set the installer Publisher to the Git Fundamentals team
+      - name: Update installer metadata for microsoft/git
         shell: bash
         run: |
           b=/usr/src/build-extra &&
-          sed -i -e 's/^\(AppPublisher=\).*/\1The Git Fundamentals Team at GitHub/' $b/installer/install.iss
+          # Remove '.vfs' from version
+          version=$(echo ${{ needs.prereqs.outputs.tag_version }} | sed s/.vfs//g) &&
```

Comment:
Is there a specific reason that `.vfs` is being removed? If the reason is "AppVersion only handles numeric characters", the filter on allowed version tags would need to be updated.

Reply:
Found it! https://github.com/git-for-windows/build-extra/blob/main/installer/release.sh#L105-L107

```
sed -e 's/\.[^0-9]*\.[^0-9]*\./\./g' \
	-e 's/\.[^.0-9]*\./\./g' -e 's/\.rc/\./g')"
```

Reply:
Sorry for the comment spam, but I'm trying to understand what's happening here. Why do you manually need to replace the version at all, if `build-extra/installer/release.sh` already sets the version in the format you want?

Reply:
(I sadly just realized that my 2 most recent comments have been 'pending' for quite some time, which explains why I wasn't getting responses 😬)